src/config.rs

```diff
@@ -26,6 +26,10 @@ pub struct Config {
     #[cfg(test)]
     pub(crate) s3_bucket_is_temporary: bool,
 
+    // CloudFront domain which we can access
+    // public S3 files through
+    pub(crate) s3_static_domain: String,
```

Comment:
For local setups it would be good to have this as `s3_static_root_path` (we can set it to `http://localhost:9000/rust-docs-rs` and configure min.io to allow downloads).

Reply:
valid point, I updated the name